functorch/csrc/BatchRulesRandomness.cpp

```diff
@@ -399,6 +425,11 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchVmapMode, m) {
   UNARY_POINTWISE_RANDOM(poisson);
   UNARY_POINTWISE_RANDOM(bernoulli);
 
+  DROPOUT_UNSQUEEZE(dropout);
+  DROPOUT_UNSQUEEZE(feature_dropout);
+  DROPOUT_UNSQUEEZE(alpha_dropout);
+  DROPOUT_UNSQUEEZE(feature_alpha_dropout);
```

Comment:
These are composite operations, right? Registering a batching rule for composite operations is generally not a good idea.

Consider the following:
```
def f(x):
  y = F.dropout2d(x)
  return y.sum()

vmap(grad(f))(x)
```
This will construct a GradTensor(BatchedTensor(x)). Calling F.dropout2d( GradTensor(BatchedTensor(x))):
- will eventually make it to at::dropout(GradTensor(BatchedTensor(x)))
- GradTensor will decompose at::dropout into its constituents and BatchedTensor will not get a chance to apply the batching rule you've defined for it.

I still don't really understand what dropout is doing, so let me go read up on that a little more